src/templates/components/topTasks/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { VamcEhr } from '@/types/drupal/vamcEhr'
+
+type VamcEhrSystem = VamcEhr['field_region_page']['field_vamc_ehr_system']
+
+type TopTasksProps = {
+  path: string
+  vamcEhrSystem: VamcEhrSystem
+  administration?: { entityId: number }
+  regionPage?: { vamcEhrSystem: VamcEhrSystem }
+  office?: { vamcEhrSystem: VamcEhrSystem }
```

Comment:
The data that will get passed into here will probably be more comprehensive, but using the lease prescriptive type (just the one field we use in each of these objects) will give us flexibility to adjust what we pass in later on as needed without TypeScript getting too angry.